<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSessionDialog</class>
 <widget class="QWidget" name="AddSessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Сеанс</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="cinema_label">
        <property name="text">
         <string>Кинотеатр</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cinema_combo_box"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hall_label">
        <property name="text">
         <string>Зал</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="hall_combo_box"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="film_label">
        <property name="text">
         <string>Фильм</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="film_combo_box"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="date_label">
        <property name="text">
         <string>Дата</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDateEdit" name="data_edit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="time_label">
        <property name="text">
         <string>Время</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QTimeEdit" name="time_edit"/>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="cinema_tool_btn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="hall_tool_btn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QToolButton" name="film_tool_btn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
